CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -52,16 +52,17 @@ public IEnumerable<TimeSheet> RetrieveTimeSheets()
                 {
                     Id = (int)timeSheet["id"],
                     TimeSheetDate = timeSheet["spent_date"].ToString(),
-                    User = new User
-                    {
-                        Id = (int)timeSheet["user"]["id"],
-                        Name = timeSheet["user"]["name"].ToString()
-                    },
+                    UserId = (int)timeSheet["user"]["id"],
                     Hours = (float)timeSheet["hours"],
                     CreatedAt = DateTime.Parse(timeSheet["created_at"].ToString()),
                     UpdatedAt = DateTime.Parse(timeSheet["updated_at"].ToString())
                 }
             ).ToList(); 
         }
     }
+
+    public class HarvestDeveloper : Developer
```

Comment:
Perhaps moving this domain object to its own file to keep consistency as we have done with the `SlackDeveloper` domain object is a good idea.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/9/commits/ef7152804378d96bf2fab1de331c73a4bac12af7